src/bp-templates/bp-nouveau/js/buddypress-nouveau.js

```diff
@@ -67,6 +67,9 @@ window.bp = window.bp || {};
 			// Legal agreement enable/disabled submit button.
 			this.enableSubmitOnLegalAgreement();
 
+			// Remove requested param from the URL for Group Invitation
```

Comment:
@milanhirapra Why do we need to remove `requested` param from JS? Can't we added that conditionally from backend.

Reply:
@dipskakadiya If we click on any notification from the notification list then notification details will appear and at that time this `groups_screen_group_admin_requests` hook is called. So, it's a reason to remove it from the JS.